### PR TITLE
added hw5

### PR DESCRIPTION
**Pull request by nNg9999**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,13 +36,19 @@
...
+    "jdenticon": "^3.0.1",
     "joi": "^17.2.1",
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.20",
     "mongodb": "^3.6.1",
     "mongoose": "^5.10.5",
     "morgan": "^1.10.0",
+    "multer": "^1.4.2",
     "password-hash": "^1.2.2",
+    "save-dev": "0.0.1-security",
```
Comment (odykyi) [inline package.json]: remove this

File: package.json [hunk 2]
```diff
@@ -36,13 +36,19 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "faker": "^5.1.0",
```
Comment (odykyi) [inline package.json]: remove this

File: src/helpers/moveFile.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const fs = require('fs-extra');
+
+async function moveFile(oldPath, newPath) {
+  await fs.move(oldPath, newPath, function (err) {
```
Comment (odykyi) [inline src/helpers/moveFile.js]: fs.move can't be in one time be as `promise` and have `callback in 3rd argument` 

you should use promise style or callback style, not both

File: src/helpers/multer.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const multer = require("multer");
+const path = require("path");
+const config = require("../../config");
+
+const storage = multer.diskStorage({
+  destination: async (req, file, cb) => {
```
Comment (odykyi) [inline src/helpers/multer.js]: remove async

File: src/helpers/multer.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    try {
+      cb(null, config.avaPath);
+    } catch (e) {
+      console.error("writefile error", e);
+    }
+  },
+  filename: async (req, file, cb) => {
```
Comment (odykyi) [inline src/helpers/multer.js]: remove async

**Review by odykyi — CHANGES_REQUESTED** [review]
comments

File: package.json (hunk 1 above)
Comment (nNg9999) [reply package.json]: corrected

File: package.json (hunk 2 above)
Comment (nNg9999) [reply package.json]: corrected

File: src/helpers/moveFile.js (hunk 3 above)
Comment (nNg9999) [reply src/helpers/moveFile.js]: corrected

File: src/helpers/multer.js (hunk 4 above)
Comment (nNg9999) [reply src/helpers/multer.js]: corrected

File: src/helpers/multer.js (hunk 5 above)
Comment (nNg9999) [reply src/helpers/multer.js]: corrected

**Pull request merged**